setup.py

```diff
@@ -1,28 +1,39 @@
 from setuptools import setup
 from os.path import basename, splitext
 from glob import glob
+import os
 from setuptools import find_packages
 
-type_geometry_requires = ["shapely", "geopandas==0.6.0"]
+
+type_geometry_requires = ["shapely", "geopandas>=0.6.0"]
 type_image_path_requires = ["imagehash", "Pillow"]
 
-install_requires = (
-    ["numpy", "pandas==0.25.1", "networkx", "tangled_up_in_unicode==0.0.3", "attr"]
-    + type_geometry_requires
-    + type_image_path_requires
-)
+install_requires = [
+    "numpy",
+    "pandas>=0.25.3",
+    "networkx",
+    "tangled_up_in_unicode>=0.0.3",
+    "shapely",
+    "imagehash",
+    "pillow",
+    "mypy",
+    "black",
+    "attr",
+]
+
+install_requires += type_geometry_requires + type_image_path_requires
```

Comment:
The geopandas  dependency is problematic on Windows, let's make it an optional requirement.